modules/server/auth/main.tf

```diff
@@ -0,0 +1,20 @@
+data "template_file" "auth_service" {
+  template = file("${path.module}/auth.service.tmpl")
+
+  vars = {
+    version = "latest"
+    auth_host = "https://auth.dev.germanbiobanknode.de"
+    mail_host = "out-cloud.mms.t-systems-service.com"
+    mail_port = "25"
```

Comment:
default

Reply:
So mail_port and mail_protocol?
What are the defaults? I looked in searchbroker 2.2.0 but cant find any

Reply:
Yes we should document the defaults in Searchbroker. But usually port 25 is the default smtp port.